code/deep_learning_compiler_run_model_example.cpp

```diff
@@ -87,41 +89,37 @@ OMTensorList* generate_input(std::vector<OMTensor *>& input_tensor_vector){
         // Use CreateWithOwnership(true) in this case so the data array is
```

Comment:
Consider updating the comment to be specific that the memory obtained from the malloc for the input_data array is freed.